src/main/java/org/trips/service_framework/utils/CmsUtils.java

```diff
@@ -44,4 +43,12 @@ private static String reformatTreatmentString(String treatment) {
         treatment = treatment.replaceAll("\"", "");
         return treatment.trim();
     }
+
+    public static boolean isNull(String str) {
+        return StringUtils.isEmpty(str) || CmsConstants.NOT_APPLICABLE.equals(str);
```

Comment:
please take care of empty treatment values as well.

Reply:
sanitise treatment method takes care all such scenarios